public/js/validate_form.js

```diff
@@ -4,77 +4,75 @@ import axios from 'axios';
 
 if (document.querySelector('.add-event-form')) {
   // add a new place
-  document
-    .getElementById('addPlaceIcon')
-    .addEventListener('click', function(e) {
-      document.getElementById('addPlaceContainer').classList.toggle('hide');
-      document.getElementById('selectAllPlaces').classList.toggle('hide');
-    });
+  document.getElementById('addPlaceIcon').addEventListener('click', e => {
+    document.getElementById('addPlaceContainer').classList.toggle('hide');
+    document.getElementById('selectAllPlaces').classList.toggle('hide');
+  });
 
-  function handleValidate(messageBox, message, input, childInputs) {
+  const handleValidate = (messageBox, message, input, childInputs) => {
     window.location.href = input;
     if (childInputs) {
-      var inputParent = document.querySelector(input).children;
+      const inputParent = document.querySelector(input).children;
       inputParent[0].style = 'border: 0.1rem red solid';
       inputParent[1].style = 'border: 0.1rem red solid';
     } else {
-      var container = document.querySelector(input);
+      const container = document.querySelector(input);
       container.style = 'border: 0.1rem red solid';
     }
     document.querySelector(messageBox).innerHTML = message;
-  }
+  };
 
-  function validated(input, messageBox, childInputs) {
+  const validated = (input, messageBox, childInputs) => {
     if (childInputs) {
-      var inputParent = document.querySelector(input).children;
+      const inputParent = document.querySelector(input).children;
       inputParent[0].style = 'border: 0.1rem gray solid';
       inputParent[1].style = 'border: 0.1rem gray solid';
     } else {
       var container = document.querySelector(input);
       container.style = 'border: 0.1rem gray solid';
     }
     document.querySelector(messageBox).innerHTML = '';
```

Comment:
Any reason for this to be innerHTML and not textContent?

Reply:
good shout didn't pay attention to this

Reply:
Good shout but not fixing?